lib/loaders/sass.js

```diff
@@ -31,7 +31,8 @@ module.exports = {
             sassLoaders.push({
                 loader: 'resolve-url-loader',
                 options: {
-                    sourceMap: webpackConfig.useSourceMaps
+                    sourceMap: webpackConfig.useSourceMaps,
+                    ...webpackConfig.sassOptions.resolveUrlLoaderOptions
```

Comment:
I don't think that the spread operator is allowed in this context with the version of ESLint we are currently using (which is why the "lint" job fails on Travis).

So two choices there:
* update ESLint (but maybe that's a chore for another PR)
* use `Object.assign()`

Reply:
the issue is not the eslint version, but the node.js version.

Reply:
The linter is executed using Node 10 which should supports this.
It should even be ok under Node 8 since that job is green: https://travis-ci.org/symfony/webpack-encore/jobs/560124277

```
$ nvm use v8.15.1
Now using node v8.15.1 (npm v6.4.1)

$ node
> const foo = { foo: 1, bar: 2 };
undefined
> const bar = { ...foo, bar: 3 };
undefined
> bar
{ foo: 1, bar: 3 }
```

Reply:
Maybe that could be solved without updating eslint though, by changing the following option of our `.eslintrc.js` to `2018`:

https://github.com/symfony/webpack-encore/blob/9a3cb43e880f10c54e8319b76433058137da78d2/.eslintrc.js#L9

Reply:
Actually you were partially right @stof, using `ecmaVersion: 2018` is triggering other errors:

```
  99:22  error  Spread properties are not supported yet on Node 8.0.0                                                           node/no-unsupported-features
  99:22  error  Rest/spread properties are not supported until Node.js 8.3.0. The configured version range is '8.* || >= 10.*'  node/no-unsupported-features/es-syntax
```

We should probably change that range since most Webpack loaders are now requiring `>= 8.9.0` (for instance: [`css-loader`](https://github.com/webpack-contrib/css-loader/pull/948), [`url-loader`](https://github.com/webpack-contrib/url-loader/pull/182), [`file-loader`](https://github.com/webpack-contrib/file-loader/blob/d441daa955246967686d24a49da83329fad6d1b0/package.json#L12), ...)

But as I said earlier, maybe it should be handled in another PR.

Reply:
I just updated the PR with changes requested.